src/applications/disability-benefits/996/form-entry.jsx

```diff
@@ -11,5 +11,6 @@ startApp({
   url: manifest.rootUrl,
   reducer,
   routes,
+  entryName: manifest.entryName,
```

Comment:
OMG! <img alt="ghe-emoji" title=":_onion_head_facepalm:" src="https://cloud.githubusercontent.com/assets/136959/13371484/cfdcf732-dcec-11e5-83f6-b4447352eb9c.gif"> 

I can't believe I forgot to include this.... LOL

And yeah, it's perfect 😺